src/liquid/components/ld-input/ld-input.tsx

```diff
@@ -293,12 +327,12 @@ export class LdInput implements InnerFocusable {
       return (
         <Host class={cl} onClick={this.handleClick}>
           <textarea
+            {...this.clonedAttributes}
```

Comment:
You could use destructuring here, to not apply the type attribute.

Reply:
Changed.